Task1BLINK.ino

```diff
@@ -0,0 +1,18 @@
+/*This sketch meets the Acceptance criteria of this task: https://trello.com/c/ba783WmU/2-blink-an-led
+*/
+ 
+void setup() {
+
```

Comment:
Phoebe you done well on this, but always try not to have consistent spacing in your Arduino coding, but Phoebe you will get the hang of this I promise you Phoebe and your own the Arduino coding. #NeverGiveupPhoebe# RisequeenPhoebe# #BeststudentthatsgoingPhoebe# #GirlPowerPhoebe#